app/screens/home/SettingScreen.tsx

```diff
@@ -708,18 +728,17 @@ const SettingScreen = (props: any): any => {
       type: DocumentPicker.types.allFiles,
     })
       .then(async (res: any) => {
-        //console.log('<<<<<importDataIOS>>>>>>', res)
+        console.log('<<<<<importDataIOS>>>>>>', res)
         let oldChildrenData: any = []
         if (res.length > 0 && res[0].uri) {
           if (res[0].name.endsWith(".json")) {
             const decryptFileContent: any = await RNFS.readFile(decodeURIComponent(res[0].uri), 'utf8').then((edata: any) => {
-              //console.log("edata", edata);
-              //console.log("encryptionsKey", encryptionsKey);
-              
+              console.log("edata", edata);
+              console.log("encryptionsKey", encryptionsKey);

```

Comment:
Is it safe to show encryptionsKey to the user? Let's remove or comment out this log.